### ISSUE TITLE
Import all styling

### ISSUE THREAD

**Issue by markoheijnen** [lanche86/wordcamp-belgrade-2016#2]
Currently styling happens in two places. Should we add all styling in the custom CSS file?

**Comment by lanche86** [comment]
@markoheijnen yeah, we will, currently Aleksandar is working on website styling as well, and it was easier for me to let him write the CSS there, and after we set everything I will add all CSS here. Sounds good?

**Comment by markoheijnen** [comment]
Sure, that works.
